docker-compose.yml

```diff
@@ -1,6 +1,24 @@
 version: '3.6'
 
 services:
+  app:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    environment:
+      - RAILS_LOG_TO_STDOUT=true
+      - POSTGRES_HOST=db
+      - SETTINGS__REDIS__HOSTNAME=redis
+      - RAILS_ENV=development
+    volumes:
+      - .:/app
```

Comment:
I'd like @jcoyne or @justinlittman to comment on the `volumes:` section, if that's OK.

Reply:
For instance, can this section handle mounting `tmp/pids` so we don't need the `.gitignore` changes above? Or do we want to share the state in this dir between local dev and the docker instance?

Reply:
Good question. I can test that. When I ran into errors related to the pids, this was the suggested route, so I didn't try mounting.